src/core/loader/get-catalogue.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+import { initCatalogue } from "../../components/shop/item-catalogue.js";
+
+export const loadCatalogue = (screen, config) => {
+    const catalogueKeys = getCatalogueKeys(config);
+
+    catalogueKeys.forEach(loadToCache(screen));
+
+    screen.load.once("complete", () =>
+        catalogueKeys.forEach(key => initCatalogue(key, screen.cache.json.get(`catalogue-${key}`))),
+    );
+
+    screen.load.start();
+};
+
+export const getCatalogueKeys = config => {
+    const catalogueKey = item => item[1].catalogueKey;
+
+    return Object.entries(config).map(catalogueKey).filter(Boolean);
```

Comment:
can you just use `Object.values` instead of using entries to get the value? (finding it hard to follow what's happening here)

Reply:
I'm guessing it could just end up as `Object.values(config).filter(Boolean);`

Reply:
5a44dfc